### PR TITLE
chore: Move runner-ct dependencies to devDependencies

### PR DESCRIPTION
**Pull request by tgriesser**
### Additional details

Looking through the dependencies in the released app, saw that there's about 75mb of `node_modules` in `runner-ct` that are likely not needed b/c they should only be used during the build phase of the frontend module.

### How has the user experience changed?

- Reduced the bundle size of the build

### PR Tasks

- [ ] Have tests been added/updated?
- [ ] Has the original issue or this PR been tagged with a release in ZenHub? 
- [ ] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [ ] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?
- [ ] Have new configuration options been added to the [`cypress.schema.json`](https://github.com/cypress-io/cypress/blob/develop/cli/schema/cypress.schema.json)?

### CODE REVIEW

File: packages/runner-ct/package.json [hunk 1]
```diff
@@ -16,23 +16,39 @@
...
+    "eslint-plugin-react": "^7.22.0",
+    "eslint-plugin-react-hooks": "^4.2.0",
     "express": "^4.17.1",
     "fuzzysort": "^1.1.4",
     "globby": "^11.0.1",
     "hotkeys-js": "3.8.2",
     "html-webpack-plugin": "^4.5.0",
     "koa": "^2.13.0",
-    "lodash": "^4.17.20",
     "mobx": "5.15.4",
```
Comment (elevatebart) [inline packages/runner-ct/package.json:52]: `mobx` is called in `src/lib/state.ts`. It should stay in the dependencies.

File: packages/runner-ct/package.json [hunk 2]
```diff
@@ -41,23 +57,9 @@
     "react-devtools-inline": "^4.10.1",
     "react-dom": "16.8.6",
     "react-split-pane": "^0.1.92",
```
Comment (elevatebart) [inline packages/runner-ct/package.json:59]: `react-split-pane` is not used anymore and should be removed from the dependencies.

**Review by elevatebart — CHANGES_REQUESTED** [review]
Hello @tgriesser,

Thank you for this PR, this will indeed be very helpful to make the package lighter. 
Do you mind if I commit on your PR?

There are a few hiccups here and there that would like to fix.

- `react` & `react-dom` are missing from the dependencies
- `mobx` is missing too
- `react-split-pane` is not used anymore and should be entirely removed from the dependencies.

File: packages/runner-ct/package.json (hunk 2 above)
Comment (tgriesser) [reply packages/runner-ct/package.json:59]: It seems it is: https://github.com/cypress-io/cypress/blob/97d5a6d88e48c48ef9bc7048a49c9651e313a085/packages/runner-ct/src/app/RunnerCt.tsx#L5

Comment (elevatebart) [reply packages/runner-ct/package.json:59]: I got too deep in the unification... Sry

File: packages/runner-ct/package.json (hunk 1 above)
Comment (tgriesser) [reply packages/runner-ct/package.json:52]: We only need server-run dependencies in the `dependencies`, the rest that are client side can be in `devDependencies`

**Review by elevatebart — APPROVED** [review]
OK the chunking is done inside of `runner-ct.ts` so only the dependencies in `runner-ct.ts` should not be `devDependencies`. The rest are.

Thank you Tim for the precision.

**Pull request merged**